### ISSUE TITLE
Qustion about the setting of max_batches

### ISSUE THREAD

**Issue by zt1112** [jkjung-avt/yolov4_crowdhuman#21]
In [AlexeyAB/darknet](https://github.com/AlexeyAB/darknet), it says that "change line max_batches to (classes*2000, but not less than number of training images and not less than 6000)".
so why set max_batches=6000, instead of more？

**Comment by jkjung-avt** [comment]
You should determine hyperparameters based on validation results and  by looking at the loss curve.  Total number of training iterations, along with learning rates and batch sizes, is one of those important hyperparameters you need to tune during training.

If you look at the [training/loss curve](https://github.com/jkjung-avt/yolov4_crowdhuman/blob/master/doc/chart_yolov4-crowdhuman-608x608.png), it appears that the loss wouldn't improve much if you train the model longer.  So I'd say "max_batches=6000" is an adequate setting.

You could try to train the model for longer, but the gain would likely be minimal.
